### PR TITLE
fix function precedence

### PR DESCRIPTION
**Pull request by logansnow17**

### CODE REVIEW

File: src/grammar-parser/grammar-parser.jison [hunk 1]
```diff
@@ -3,17 +3,17 @@
 %lex
 %%
 \s+                                                                                             {/* skip whitespace */}
+[A-Za-z]{1,}[A-Za-z_0-9\.]+(?=[(])                                                              {return 'FUNCTION';}
```
Comment (logansnow17) [inline src/grammar-parser/grammar-parser.jison:6]: Moved the precedence for functions above the precedence for lowercase 'e'

Comment (logansnow17) [reply src/grammar-parser/grammar-parser.jison:6]: Otherwise lowercase exp was failing but uppercase EXP worked

Comment (TimothyWang56) [reply src/grammar-parser/grammar-parser.jison:6]: yeah this makes sense

**Review by TimothyWang56 — APPROVED** [review]
Changes make sense. Verified that Logan ran the tests locally and were successful. I also checked that there were `e` tests present to validate that they were still parsed correctly with this precedence reorder

**Pull request merged**